pkg/convert/registryv1.go

```diff
@@ -291,6 +291,14 @@ func Convert(in RegistryV1, installNamespace string, targetNamespaces []string)
 	}
 
 	objs := []client.Object{}
+
+	// Add the install namespace
+	ns := &corev1.Namespace{
+		TypeMeta:   metav1.TypeMeta{Kind: "Namespace", APIVersion: "v1"},
+		ObjectMeta: metav1.ObjectMeta{Name: installNamespace},
+	}
+	objs = append(objs, ns)
```

Comment:
I _just_ removed this in the `InstallNamespace` PR. The expectation is that the install namespace will already exist.

Reply:
?I think that wasn't true in our PoC, but it means we'll have to deal with it better during test.

Reply:
It wasn't. This change just merged yesterday in rukpak and today in operator-controller. If you're able to rebase to operator-controller `main`, you shouldn't need this.